cmd/corectl/main.go

```diff
@@ -79,7 +79,10 @@ func configGenerator(db *sql.DB, args []string) {
 
 	var flags flag.FlagSet
 	maxIssuanceWindow := flags.Duration("w", 24*time.Hour, "the maximum issuance window `duration` for this generator")
-	isSigner := flags.Bool("s", false, "whether this core is a signer")
+	flagK := flags.String("k", "", "local `pubkey` for signing blocks")
```

Comment:
I copied the help string format from configNongenerator. Not sure about the previous history - do we want to change the format throughout?